docroot/themes/humsci/humsci_basic/readme.md

```diff
@@ -106,30 +106,32 @@ _Note: Differences in content will also be reported as failures. Some failures c
 To make it easier to find work being done we should use the following naming conventions:
```

Comment:
The whole "Contributing" section doesn't make sense anymore, as all contributions are done to the platform as a whole and not individually to the theme. It's better to remove this section altogether.